src/phases.ts

```diff
@@ -66,6 +64,7 @@ import { Species } from "#enums/species";
 import { TrainerType } from "#enums/trainer-type";
 import { applyChallenges, ChallengeType } from "./data/challenge";
 import { pokemonEvolutions } from "./data/pokemon-evolutions";
+import { Stat, BattleStat, getStatKey, getStatStageChangeDescriptionKey } from "#enums/stat";
```

Comment:
```suggestion
import { Stat, type BattleStat, getStatKey, getStatStageChangeDescriptionKey } from "#enums/stat";
```
NIT: Label types and standardize order